### DISCUSSION TITLE
单双页的页边距不一样？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#290] [Q&A] [answered]
您好，

貌似单双页页边距不一样。
![image](https://github.com/whutug/whu-thesis/assets/45897471/3f0f4e99-393f-42e5-bb12-180bb272ffd1)

是我使用的问题，还是模板特定设置成这样？

谢谢！

**Comment by SwitWu** [comment]
双面打印是这样的

**Reply by earthmoon** [reply]
您好，
这个参数在cls文件的哪个位置？我去把它先调整过来，因为目前主要使用电子版，居中对齐更好。
谢谢！

**Reply by SwitWu** [reply]
`\documentclass[oneside]{whu-thesis}`

**Reply by earthmoon** [reply]
您好，
有没有办法既保留双面，又能调整到居中对齐？twoside关于这个的参数在哪呢。
谢谢！

**Reply by SwitWu** [reply, marked as answer]
https://github.com/whutug/whu-thesis/blob/a03ab5a275c84afdd4085d8544bcdf8b313e58b7/whu-thesis.cls#L434-L471

修改 `left` 和 `right` 的值

**Reply by earthmoon** [reply]
您好，
我想了下，如果设置一个print参数，当生成print版时，对这个页边距进行外移是不是更好？
谢谢！

**Reply by SwitWu** [reply]
实现不难，但是基本没人有这个需求

**Reply by earthmoon** [reply]
您好，
确实也是这样，很少有人追求这种细节。
谢谢！
